samples/bluetooth/mesh/light/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application overlay - nrf5340dk secure image
+
+CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=n
```

Comment:
Why was it set to `n`?

Reply:
I copied the option from nrf5340dk_nrf5340_cpuapp_ns.conf. It was set from before, so I guessed it was done for a reason.